editor.planx.uk/src/@planx/components/Send/uniform/xml.ts

```diff
@@ -42,6 +42,10 @@ export function makeXmlString(
       <common:FileName>application.csv</common:FileName>
       <common:Reference>Other</common:Reference>
     </common:FileAttachment>
+    <common:FileAttachment>
+      <common:FileName>boundary.geojson</common:FileName>
+      <common:Reference>Other</common:Reference>
+    </common:FileAttachment>
```

Comment:
It would also then be useful to have a test case for this in `xml.test.ts`, so that when we merge https://github.com/theopensystemslab/planx-new/pull/1220 we can be sure that the logic has been correctly implemented there.